CMakeLists.txt

```diff
@@ -563,7 +563,7 @@ function(install_kdb FILE_NAME COMPONENT_NAME)
     if(__contents MATCHES "version")
         list(APPEND LFS_MISSING_FILES ${__fname})
     else()
-        unpack_db(${FILE_NAME})
+        unpack_db(${FILE_NAME} FILE_SUFFIX)
```

Comment:
[Recommendation]
```suggestion
        unpack_db(${FILE_NAME} UNUSED_FILE_SUFFIX)
```

Reply:
The design of this stuff is a bit clumsy, so I would also recommend adding a comment after line 532:
```
if(NOT MIOPEN_USE_SQLITE_PERFDB AND __extension STREQUAL ".db")
```
that should inform the reader that this branch is dedicated to a special case of unpacking system-find-db files converted to text format.

/cc @DrizztDoUrden

Reply:
UPD: system-perf-db files

Reply:
@atamazov should we close this PR and use #2802 instead? That's my plan